src/Model/Setting/BookkeepingSetting.php

```diff
@@ -0,0 +1,549 @@
+    {
+        return $this->factoringReceivableAccount;
+    }
+
+    /**
+     * @param string $factoringReceivableAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withFactoringReceivableAccount(string $factoringReceivableAccount)
+    {
+        $new = clone $this;
+        $new->factoringReceivableAccount = $factoringReceivableAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getNonAllocatedAccount(): string
+    {
+        return $this->nonAllocatedAccount;
+    }
+
+    /**
+     * @param string $nonAllocatedAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withNonAllocatedAccount(string $nonAllocatedAccount)
+    {
+        $new = clone $this;
+        $new->nonAllocatedAccount = $nonAllocatedAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getIncomePayoutAccount(): string
+    {
+        return $this->incomePayoutAccount;
+    }
+
+    /**
+     * @param string $incomePayoutAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withIncomePayoutAccount(string $incomePayoutAccount)
+    {
+        $new = clone $this;
+        $new->incomePayoutAccount = $incomePayoutAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getWrittenDownReceivablesAccount(): string
+    {
+        return $this->writtenDownReceivablesAccount;
+    }
+
+    /**
+     * @param string $writtenDownReceivablesAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withWrittenDownReceivablesAccount(string $writtenDownReceivablesAccount)
+    {
+        $new = clone $this;
+        $new->writtenDownReceivablesAccount = $writtenDownReceivablesAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getExpectedLossAccount(): string
+    {
+        return $this->expectedLossAccount;
+    }
+
+    /**
+     * @param string $expectedLossAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withExpectedLossAccount(string $expectedLossAccount)
+    {
+        $new = clone $this;
+        $new->expectedLossAccount = $expectedLossAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return array
+     */
+    public function getRegionalSweden(): array
+    {
+        return $this->regionalSweden;
+    }
+
+    /**
+     * @param array $regionalSweden
+     *
+     * @return BookkeepingSetting
+     */
+    public function withRegionalSweden(array $regionalSweden)
+    {
+        $new = clone $this;
+        $new->regionalSweden = $regionalSweden;
+
+        return $new;
+    }
+
+    /**
+     * Create an API response object from the HTTP response from the API server.
+     *
+     * @param array $data
+     *
+     * @return self
+     */
+    public static function createFromArray(array $data)
+    {
+        return null;
```

Comment:
Absolute! I left it empty because i began with the creation of the structure of the model . it's fixed now.